modules/gdscript/gdscript_vm.cpp

```diff
@@ -1776,7 +1776,14 @@ Variant GDScriptFunction::call(GDScriptInstance *p_instance, const Variant **p_a
 							}
 						}
 					}
-					err_text = _get_call_error(err, "function '" + methodstr + (is_callable ? "" : "' in base '" + basestr) + "'", (const Variant **)argptrs);
+
+					String tool_suffix;
+					if (base->has_method(methodstr)) {
```

Comment:
I think this is too broad, maybe:
```suggestion
					if (err.error == Callable::CallError::CALL_ERROR_INVALID_METHOD && base->has_method(methodstr)) {
```
Same below